catalog-rest-service/src/main/resources/json/schema/entity/services/connections/database/athenaConnection.json

```diff
@@ -73,5 +73,6 @@
       "$ref": "../connectionBasicType.json#/definitions/supportsProfiler"
     }
   },
-  "additionalProperties": false
+  "additionalProperties": false,
+  "required": ["s3StagingDir"]
```

Comment:
is awsConfig required as well? Or can this potentially work without it?

Reply:
Also workgroup might be necessary. Just tested it without it and it did not work. Please, use this issue as a chance to test these different points manually and make sure that the resulting JSON Schema reflects the reality. Otherwise we won't stop having to open fixes about the same.
Thanks!

Reply:
as discussed, we could try to remove the database as well from the schema